package.json

```diff
@@ -9,7 +9,7 @@
         "build": "npm-run-all clean build:angular webpack build:icons",
         "pretest": "webpack",
         "test": "karma start ./karma.conf.js",
```

Comment:
Does it make sense to add it here also and execute with npm-run-all?

Reply:
You don't want this to run locally, so adding it here would fail the test command locally.